tool/src/org/antlr/v4/codegen/Target.java

```diff
@@ -57,17 +54,14 @@ protected Target(CodeGenerator gen, String language) {
 		targetCharValueEscape['\''] = "\\'";
 		targetCharValueEscape['"'] = "\\\"";
 		this.gen = gen;
-		this.language = language;
 	}
 
+	protected abstract TargetType getTargetType();
```

Comment:
Hmm... Maybe Target language is better than target type?

Reply:
Ok, `targetLanguage`. In theory, it could be a platform (.NET, JVM) but let's leave it for the future.

Reply:
Maybe just `language` instead of `targetLanguage` for clarity (it's clear what this language means)?